testing/setup/rbac.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can we source this from KIT instead?

Reply:
This is a little different. These include SA and RBAC needed for the tekton pods. We should dictate how our management cluster is setup, but leave the add-ons to KIT where possible. This will change the setup.sh script soon.